anomalib/deploy/inferencers/openvino.py

```diff
@@ -20,7 +20,8 @@
 import cv2
 import numpy as np
 from omegaconf import DictConfig, ListConfig
-from openvino.inference_engine import IECore  # pylint: disable=no-name-in-module
+
+# from openvino.inference_engine import IECore  # pylint: disable=no-name-in-module
```

Comment:
It's unfortunately not possible to comment out this import since openvino stuff depends on this. Perhaps, we could add a check if openvino is installed or not.

Reply:
This is no longer an issue on mac.